app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/EmailSignInSignUpFragment.kt

```diff
@@ -207,8 +199,8 @@ class EmailSignInSignUpFragment : Fragment() {
     private fun updateUI(status: STATUS) {
         when (status) {
             STATUS.SignedIn -> {
-                binding.signIn.isEnabled = false
-                binding.signUp.isEnabled = false
+                binding.signIn.isEnabled = true
+                binding.signUp.isEnabled = true
```

Comment:
why are these inverted? should the sign out button be disabled in this case then?

Reply:
Thanks Robert. They are not intended changes. They are legacy trace of release test execution.